docker-compose.yaml

```diff
@@ -63,7 +63,7 @@ services:
       - 5555
     command: |
       bash -c "
-        enclave_manager --lmdb_url http://avalon-lmdb:9090
+        enclave_manager --lmdb_url http://avalon-lmdb:9090 --worker_id singleton-worker-1
```

Comment:
Need not mention explicitly. Neither in any of the other docker-compose files. They will pick up the default. Need to pass here as well when intention is to override the one in the enclave manager config file. Rather update in the single manager config toml. That would suffice.

Reply:
Makes sense to put it in config file as it would avoid changes at many places. Will do.